TTDQueries/README.md

```diff
@@ -48,6 +48,6 @@ Check out [TTD - Queries] to learn more.
 * [Defrag Tools - Time Travel Debugging - Advanced](https://channel9.msdn.com/Shows/Defrag-Tools/Defrag-Tools-186-Time-Travel-Debugging-Advanced)

```

Comment:
these links need to be updated:


[https://learn.microsoft.com/en-us/shows/defrag-tools/185-time-travel-debugging-introduction](https://learn.microsoft.com/en-us/shows/defrag-tools/185-time-travel-debugging-introduction)

[https://learn.microsoft.com/en-us/shows/defrag-tools/186-time-travel-debugging-advanced](https://learn.microsoft.com/en-us/shows/defrag-tools/186-time-travel-debugging-advanced)